crates/uv/tests/lock_scenarios.rs

```diff
@@ -1413,8 +1413,8 @@ fn fork_marker_inherit_combined_disallowed() -> Result<()> {
             "implementation_name != 'cpython' and implementation_name != 'pypy' and sys_platform == 'darwin'",
         ]
         dependencies = [
-            { name = "package-b", version = "1.0.0", source = { registry = "https://astral-sh.github.io/packse/PACKSE_VERSION/simple-html/" }, marker = "implementation_name == 'pypy'" },
-            { name = "package-b", version = "2.0.0", source = { registry = "https://astral-sh.github.io/packse/PACKSE_VERSION/simple-html/" }, marker = "implementation_name == 'cpython'" },
+            { name = "package-b", version = "1.0.0", source = { registry = "https://astral-sh.github.io/packse/PACKSE_VERSION/simple-html/" }, marker = "implementation_name == 'pypy' and sys_platform == 'darwin'" },
+            { name = "package-b", version = "2.0.0", source = { registry = "https://astral-sh.github.io/packse/PACKSE_VERSION/simple-html/" }, marker = "implementation_name == 'cpython' and sys_platform == 'darwin'" },
```

Comment:
These may not be strictly necessary... Because you can only reach this node on `sys_platform == 'darwin'`.

Reply:
(Yeah confirmed. Is it wrong to include then?)

Reply:
I don't think it's _wrong_, but it does seem superfluous. But I'm okay to run with it for now.